packages/core/src/dialects/ibmi/query-generator-typescript.ts

```diff
@@ -33,6 +38,65 @@ export class IBMiQueryGeneratorTypeScript extends AbstractQueryGenerator {
     ]);
   }
 
+  addConstraintQuery(tableName: TableNameOrModel, options: AddConstraintQueryOptions) {
+    return joinSQLFragments([
+      'ALTER TABLE',
+      this.quoteTable(tableName),
+      'ADD',
+      this.getConstraintSnippet(tableName, options),
+    ]);
+  }
+
+  removeConstraintQuery(tableName: TableNameOrModel, constraintName: string, options?: RemoveConstraintQueryOptions) {
```

Comment:
Same here, this could be moved to abstract-query-generator & use the dialect feature flags to build the `REMOVE_CONSTRAINT_QUERY_SUPPORTED_OPTIONS` array for the current dialect

Reply:
Done